metadata.rb

```diff
@@ -10,9 +10,9 @@
 
 depends          'base'
 depends          'build-essential'
-depends          'git', '~> 4.3'
+depends          'git'
 depends          'java'
-depends          'jenkins', '~> 2.4.0'
+depends          'jenkins', '~> 5.0.0'
 depends          'osl-haproxy'
 depends          'ssh-keys'
```

Comment:
I have always wondered why don't we lock down all versions of the cookbooks which are from upstream? If they make a change which is undesirable to our infra, we might come to know if only once things start crashing.

I understand this probably delves a bit into how we do DevOps and would like to understand how we aim to manage such a risk.